source/global.h

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
I would expect the default/standard ES copyright header here.

Reply:
ok